test/object.go

```diff
@@ -33,6 +33,7 @@ func Object[T client.Object](base T, overrides ...T) T {
 	for _, src := range append([]T{base}, overrides...) {
 		lo.Must0(mergo.Merge(dest, src, mergo.WithOverride))
 	}
+	dest.SetCreationTimestamp(metav1.Now())
```

Comment:
This should be set when applied by the API Server

Reply:
It's not -- I tried it and I think because we are using a mock it doesn't get instantiated.